src/graphnet/data/sqlite_dataset.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         truth_table: str = 'truth',
         selection: Optional[List[int]] = None,
         dtype: torch.dtype = torch.float32,
+        node_representation = 'pulse',
```

Comment:
This would probably be best suited for an "enum" or similar type of structure (i.e., to have a fixed set of options), but this is perfectly fine for now.